ext/rav1e.cmd

```diff
@@ -11,11 +11,10 @@
 : # Also, the error that "The target windows-msvc is not supported yet" can safely be ignored provided that rav1e/target/release
 : # contains rav1e.h and rav1e.lib.
 
-git clone -b 0.3 --depth 1 https://github.com/xiph/rav1e.git
+git clone -b 0.4 --depth 1 https://github.com/xiph/rav1e.git
 
 cd rav1e
-cargo install cbindgen
-cbindgen -c cbindgen.toml -l C -o target/release/include/rav1e/rav1e.h --crate rav1e .
+cargo install cargo-c
 
-cargo build --lib --release --features capi
+cargo cbuild --release
```

Comment:
If you want to use cargo-c you should pass `--library-type=static` to not build an unnecessary dynamic library

Reply:
It seems to be `--library-type=staticlib` instead of `--library-type=static`. Anyway, thanks for pointing that out!